src/constants/upload_types.js

```diff
@@ -0,0 +1,6 @@
+const UPLOAD_TYPES = {
+    SELF: 'self',
+    CROSS: 'cross'
+};
+
+export default UPLOAD_TYPES;
```

Comment:
Same as src/constants/index.jsx. I we can add a blank line at the end of the file to keep things consistent with the rest of our files.